src/GameSaves/OriginalGameSaveToFileWriter.cs

```diff
@@ -777,7 +785,7 @@ public void WriteSaveGame(GameSave gameSave)
                     bw.Write((byte)0);
                 }
 
-                // TODO: AI opponents
+                // TODO: AI opponents (shoulnd't be - 1?)
```

Comment:
It should be -2, because Barbarians are counted as a player, and both the Human player and the Barbarians are not handled as opponents in the original game.